src/pages/VaultsPage/CreateForm.tsx

```diff
@@ -136,14 +136,15 @@ export const CreateForm = ({
       onChange({ target: { value: undefined } });
     } else if (vaultType == 'simple') {
       setDisplayCustomToken(false);
-      if (symbol == 'USDC') {
+      if (
```

Comment:
Hmm, can we find a way to avoid repeating the list of tokens here?

Reply:
hows about this ?